### ISSUE TITLE
3.12 webui fails soon after login

### ISSUE THREAD

**Issue by LaravelFreelancerNL** [arangodb/arangodb#20842]
## My Environment

* __ArangoDB Version__:       3.12.0-1
* __Deployment Mode__:       Single Server
* __Deployment Strategy__:   Manual Start
* __Configuration__:               default, no root pw (local dev)
* __Infrastructure__:               own 
* __Operating System__:        Windows 11 with WSL/Ubuntu 20.04
* __Total RAM in your machine__:        64GB
* __Disks in use__:        SSD 
* __Used Package__:            Ubuntu .deb 

## Component, Query & Data

__Affected feature__:
web interface

## Steps to reproduce
1. Purge arangodb 3.11 (or just update, the result is the same)
2. Install ArangoDB 3.12 within WSL2, no root pw setting
3. Login with root 
4. Select _system db
5. wait for the fails

__Problem__:
Several errors occur:
1. DB: could not fetch database
2. Network tab shows several requests failing:
- job (net::Err..)
- user (net::Err..)
- root (net::Err..)
- fontawesome webfonts
3. click on databases:
-  Error shown: Failure: got unexpected server response: 11: not authorized to execute this request
- databaseDefaults request fails (401)
- database requests fails (401)

__Expected result__:
The UI to work properly.

I've posted a video of the sequence described here in the general slack channel. (The video was too large to attach to this issue).

(upon purging 3.12 and reverting to 3.11 everything works fine)

**Comment by LaravelFreelancerNL** [comment]
I've just given 3.12-1 a try and this issue is resolved.

**Comment by dothebart** [comment]
Thanks!

**Comment by LaravelFreelancerNL** [comment]
@dothebart apparently I spoke too soon. Although the behaviour has been improved a lot.

I still get similar behaviour but after a few hours instead of right away. After some time I get an authorization error and am redirected to the login screen. This has happened several times during active use (and from the looks of it during inactive use as well, but I don't know if that received the same error).

It looks like there is a client site timeout which doesn't take the DB connection and/or recent activity in to account.

**Comment by dothebart** [comment]
ok, can you take a look into the logfiles to see wether the PID changes in that range of time? Its the value in `[]` .

**Comment by LaravelFreelancerNL** [comment]
Sprry, I missed your comment. I'll check the logs when it happens again.

**Comment by LaravelFreelancerNL** [comment]
Yes it does. ArangoDB restarts in the background. There's no error in the logs before the restart.

**Comment by dothebart** [comment]
no `FATAL` logmessages at all?
